langchain/schema.py

```diff
@@ -360,7 +360,7 @@ def dict(self, **kwargs: Any) -> Dict:
         return output_parser_dict
 
 
-class OutputParserException(Exception):
+class OutputParserException(ValueError, Exception):
```

Comment:
What is this needed for?

Reply:
I added the ValueError as a base class to OutputParserException to avoid breaking code that was relying on ValueError as a way to catch exceptions from the agent. So catching ValuError still works. Not sure if this is a good idea  ?

Reply:
good idea imo. but don't need to inherit from Exception in that case (ValueError already subclass of Exception)

Reply:
true didn't think about it.